src/txns/balance.rs

```diff
@@ -47,24 +47,36 @@ async fn print_balance(pubkeys: &[PublicKey]) -> Result {
     // sample the first pubkey to determine network
     let network = pubkeys[0].network;
 
-    let client = Client::new_with_base_url(api_url(network));
```

Comment:
Oddly, this is not a diff with master...

Reply:
I guess it's probably because the PR had this diff with the original master? Weird